docs/source-pytorch/accelerators/hpu_troubleshoot.rst

```diff
@@ -0,0 +1,38 @@
+:orphan:
+
+.. _hpu_troubleshoot:
+
+Accelerator: HPU training
+=========================
+**Audience:** Gaudi chip users looking to troubleshoot their model changes during training with HPU.
+
+----
+
+Runtime Errors
+--------------
+
+This section provides troubleshooting instructions that can be referred to for common issues when training PyTorch models. The following are common functional issues that may occur when running on HPU and not on CPU/GPU.
+
+The following table outlines some possible runtime errors:
+
++----------------------------------+----------------------------------+
+| Error                            | Solution                         |
++==================================+==================================+
```

Comment:
The checks whether HPU devices are available etc. can be put into the accelerator connector (we should already have it). The error messages can be adjusted to give clear information and instructions to the user (I believe we already have this, but can be improved if needed). A troubleshooting guide like the one in this table should not be needed. I strongly advise against it, and instead resolve all issues directly through good error messages.